.github/workflows/tag.yml

```diff
@@ -13,35 +13,47 @@ jobs:
     timeout-minutes: 10
     name: Build Docker
     steps:
-      #- name: Get latest tag
-      #  id: vars
-      #  run: echo ::set-output name=tag::${GITHUB_REF:10}
       - name: Clone repository
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-        #with:
-        #  ref: ${{ steps.vars.outputs.tag }}
+
       - name: Get Latest Tag
         id: vars
         uses: WyriHaximus/github-action-get-previous-tag@v1
+
+      - name: Parse semver string
+        id: semver 
+        uses: booxmedialtd/ws-action-parse-semver@v1
+        with:
+          input_string: ${{ steps.vars.outputs.tag }}
+
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v1
         with:
           image: tonistiigi/binfmt:latest
           platforms: arm64,arm
+
       - name: Set up Docker BuildX
         uses: docker/setup-buildx-action@v1
+
       - name: Login to DockerHub
         uses: docker/login-action@v1 
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+          
       - name: Build and push
         uses: docker/build-push-action@v2
         with:
           context: .
           platforms: linux/386,linux/amd64,linux/arm64,linux/arm/v6,linux/arm/v7
           push: true
           build-args: VERSION=${{ steps.vars.outputs.tag }}
-          tags: mondoolabs/mondoo:${{ steps.vars.outputs.tag }},mondoolabs/mondoo:latest,mondoo/client:${{ steps.vars.outputs.tag }},mondoo/client:latest        
+          tags: |
+            mondoolabs/mondoo:${{ steps.vars.outputs.tag }}
```

Comment:
are the commas no longer needed? The docs say list or CSV. I would think this is new line separated

Reply:
their definition of a list is a newline-delimited string. See [here](https://github.com/docker/build-push-action#inputs)